app/models/miq_group.rb

```diff
@@ -61,7 +61,7 @@ def settings=(new_settings)
   end
 
   def self.with_roles_excluding(identifier)
-    where.not(:id => MiqGroup.joins(:miq_product_features)
+    where.not(:id => MiqGroup.unscoped.joins(:miq_product_features)
```

Comment:
How do we know we want to remove the scopes here unconditionally?

Reply:
So this is saying, "bring back all groups except those associated with the product feature of `'everything'`"

@bdunne very good question. That had slipped my mind.

In this case, I think it is best to not have `in_my_region`.
This filter is removing entries. If a user asked for all users in all regions, we would want to filter out records in other regions as well.
If we add that scope we would have `ALL_GROUPS_ALL_REGIONS - SUPER_GROUPS_MY_REGION` - Mis-displaying SUPER_GROUPS_OTHER_REGIONS`

though this is all probably a stretch

Reply:
@jrafanie ok - I changed to only remove the `select` portion since that is what was blowing up